exercises/alphametics/alphametics_test.exs

```diff
@@ -0,0 +1,162 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("alphametics.exs", __DIR__)
+end
+
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
+
+defmodule AlphameticsTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "puzzle with three letters" do
+    puzzle = "I + BB == ILL"
+
+    solution = %{
+      ?I => 1,
```

Comment:
We could also have used strings instead of raw chars as keys here. No strong benefits other than being more enumerable, as in `?A..?Z`.

Reply:
I like the character literals! It's always nice to use things like this for folks that might not have seen them before.